tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/GitHelper.cs

```diff
@@ -90,6 +91,19 @@ public async Task<string> GetRepoOwnerNameAsync(string path, bool findUpstreamPa
             throw new InvalidOperationException("Unable to determine repository owner.");
         }
 
+        // Get the full name of github repo in the format of "{owner/name}", e.g. "Azure/azure-rest-api-specs"
+        public async Task<string> GetGitHubRepoFullNameAsync(string path, bool findUpstreamParent = true)
```

Comment:
should this be internal instead or public?
and a nit: I see the other functions on this class don't mention where the repository is located at. so `GetRepoFullNameAsync` might be enough

Reply:
Aligned to other functions without 'Github' in the name.